cmd/ttn-lw-stack/commands/redis.go

```diff
@@ -0,0 +1,110 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+	nsredis "go.thethings.network/lorawan-stack/v3/pkg/networkserver/redis"
+	"go.thethings.network/lorawan-stack/v3/pkg/redis"
+)
+
+func rangeRedisKeysIteration(cl *redis.Client, cursor uint64, scanKey string, f func(k string) bool) (uint64, error) {
+	ks, cursor, err := cl.Scan(cursor, scanKey, 1).Result()
+	if err != nil {
+		return 0, err
+	}
+	for _, k := range ks {
+		if !f(k) {
+			return 0, nil
+		}
+	}
+	return cursor, nil
+}
+
+func rangeRedisKeys(cl *redis.Client, scanKey string, f func(k string) bool) error {
+	cursor, err := rangeRedisKeysIteration(cl, 0, scanKey, f)
+	if err != nil {
+		return err
+	}
+	for cursor > 0 {
+		cursor, err = rangeRedisKeysIteration(cl, cursor, scanKey, f)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+var (
+	nsCommand = &cobra.Command{
+		Use:     "ns",
+		Aliases: []string{"ns-db"},
+		Short:   "Manage Network Server",
```

Comment:
You may not agree with our historical choices for naming commands, but I think we can all agree that we prefer our commands to be consistent. So please instead write:

```
Use:   "ns-db",
Short: "Manage the Network Server database",
```